<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Command Center</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="bSend">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bConnect">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="StreamerTextEdit" name="teCommand">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>345</width>
      <height>86</height>
     </rect>
    </property>
   </widget>
   <widget class="VLCVideoWidget" name="video_container" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>511</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblFormat">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>251</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>URL Format: 'udp/h264://@:&lt;port&gt;'</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionServer_Properties"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionServer_Properties">
   <property name="text">
    <string>Server Properties</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StreamerTextEdit</class>
   <extends>QTextEdit</extends>
   <header>UI/streamertextedit.h</header>
  </customwidget>
  <customwidget>
   <class>VLCVideoWidget</class>
   <extends>QWidget</extends>
   <header>UI/vlcvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
